Scripts/script-SendMessageToOnlineUsers.yml

```diff
@@ -0,0 +1,72 @@
+commonfields:
+  id: SendMessageToOnlineUsers
+  version: -1
+name: SendMessageToOnlineUsers
+script: |-
+  var commandAvailable = function(command) {
+      var ret = false;
+      var allModules = getModulesForCommand(command);
+      if (allModules) {
+          var instanceKeys = Object.keys(allModules);
+          instanceKeys.forEach(function (key) {
+              if (allModules[key].state === 'active') {
+                  ret = true;
+              }
+          });
+      }
+      return ret;
+  };
+  var slackEnabled = commandAvailable('slack-send');
+  var emailEnabled = commandAvailable('send-mail');
```

Comment:
Do you want to return here an error in case the medium selected is not enabled?

Reply:
If it is all - than I'll only send to the enabled ones. If it is specific - there will be an error anyway.
I will add an error if none of the mediums are available.

Reply:
But you will have multiple errors of the same type, which will only create confusion.
So why not create one error here (**not** for case of **all**)

Reply:
got you. agree. added.